pkgs/development/python-modules/tensorflow/default.nix

```diff
@@ -295,9 +352,9 @@ let
     fetchAttrs = {
       # cudaSupport causes fetch of ncclArchive, resulting in different hashes
       sha256 = if cudaSupport then
-        "10m6qj3kchgxfgb6qh59vc51knm9r9pkng8bf90h00dnggvv8234"
+        lib.fakeHash
```

Comment:
Why does it have to be `lib.fakeHash`? I understand that for darwin you don't need this (and won't hit this condition due to the assert anyway) but for other platforms I assume it should be a real hash?

Reply:
I'm starting to test this on linux now, but any help will be appreciated